src/Sulu/Bundle/MediaBundle/Resources/js/types.js

```diff
@@ -17,3 +17,9 @@ export type DisplayOption =
     | 'leftBottom'
     | 'bottom'
     | 'rightBottom';
+
+export type MediaType =
+    | 'audio'
+    | 'document'
+    | 'image'
+    | 'video'
```

Comment:
Semicolon is missing.

Reply:
@alexander-schranz Is it possible to somehow extend these types? Or is our system somehow limited to these 4? Because if they are not I would not introduce a type for them, but simply use strings.

Reply:
Its possible to extend them so we should not check for this types here in the frontend.